### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by ghost**
Updated navbar. Added React Router, React Icons. Need a logo and styling.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Navbar from "./components/Navbar";
 // import "./App.css";
-// import { NavBar } from "./components/Navigation";
```
Comment (asjoss) [inline src/App.js]: Can Kerry's orginal nav bar component be kept be integrated in some way?

Comment (KerryCX) [reply src/App.js]: I am thinking, after the other pull request, I check this and then merge my code in

Comment (asjoss) [reply src/App.js]: ah right 👍

**Review by KerryCX — APPROVED** [review]
This looks safe to merge. Advise merge this so I can add my code for navigation bar.

**Comment by KerryCX** [comment]
Abul, I think this is missing "react-icons/fa"; so I am getting an error of: Compiled with problems:X

ERROR in ./src/components/CartIcon.jsx 5:0-48

Module not found: Error: Can't resolve 'react-icons/fa' in '/Users/kerryclements/Workspace/WebDev/secondhand-movies/src/components'

**Review by KerryCX — CHANGES_REQUESTED** [review]
Abul, I think this is missing "react-icons/fa"; so I am getting an error of: Compiled with problems:X

ERROR in ./src/components/CartIcon.jsx 5:0-48

Module not found: Error: Can't resolve 'react-icons/fa' in '/Users/kerryclements/Workspace/WebDev/secondhand-movies/src/components'

**Pull request merged**